client/pages/receitas/index.vue

```diff
@@ -1,9 +1,37 @@
 <template>
```

Comment:
This whole file is basically a copy of 'produtos', same for receitas/[id].

I realized I can make two components to be used on both pages, but it's not the current priority.